### PR TITLE
Add event time to write complete event

### PR DESCRIPTION
**Pull request by highker**
Record the event time when a write completes

### CODE REVIEW

**Comment by electrum** [comment]
Actually, it might be easiest to add it here. Do it the same as `HttpRequestEvent`:
```java
@EventField(fieldMapping = TIMESTAMP)
public Instant getTimeStamp()
{
    return timeStamp;
}
```

**Comment by highker** [comment]
@electrum, thanks for pointing out the mapping; scribe takes a limited number of types. If it's ok, I will just make timestamp a string.

**Comment by highker** [comment]
As discussed, we are going to refactor swift-scribe to include event metadata. Fwiw, the field name may only use "host", "timestamp", and "uuid" (all in lower cases) as airlift EventTypeMetadata generates the field names by calling "fieldName = eventField.fieldMapping().getFieldName();"

**Comment by electrum** [comment]
No, it needs to be an Instant. I believe there is validation that enforces this.

**Comment by highker** [comment]
@electrum, changed to Instant. Somehow I did a search in airlift and didn't find a place to prevent TIMESTAMP to be a String or DateTime type. When a postEvent() is called, the field will be serialized into json. Either Instant, DateTime, or String type are supported (check EventJsonSerializer and EventDataType). If no such field mapping is provided, a DateTime is used to create a default timestamp.

**Comment by dain** [comment]
@electrum can you approve your the changes?

File: presto-hive/src/main/java/com/facebook/presto/hive/WriteCompletedEvent.java [hunk 1]
```diff
@@ -66,12 +69,13 @@ public WriteCompletedEvent(
...
         this.writerImplementation = requireNonNull(writerImplementation, "writerImplementation is null");
         this.prestoVersion = requireNonNull(prestoVersion, "prestoVersion is null");
-        this.serverAddress = requireNonNull(serverAddress, "serverAddress is null");
+        this.host = requireNonNull(serverAddress, "serverAddress is null");
         this.principal = principal;
         this.environment = requireNonNull(environment, "environment is null");
         this.sessionProperties = requireNonNull(sessionProperties, "sessionProperties is null");
         this.bytes = bytes;
         this.rows = rows;
+        this.timestamp = Instant.now();
```
Comment (electrum) [inline presto-hive/src/main/java/com/facebook/presto/hive/WriteCompletedEvent.java]: Initialize this in the field declaration

**Pull request merged**